src/utils/buildThemeForm.ts

```diff
@@ -0,0 +1,15 @@
+import FormData from 'form-data';
+import { createReadStream } from 'fs';
+
+/**
+ * Build a ThemeData object from a theme zip file path
+ *
+ * @param themePath An absolute path to a theme zip file
+ * @returns ThemeData object with the theme zip file
+ */
+export const buildThemeForm = (themePath: string): FormData => {
+  const themeForm = new FormData();
+  themeForm.append('theme', createReadStream(themePath));
```

Comment:
Please refactor with my comments from before - but this is also incorrect. The key you have put here is `theme`, it should be `file` according to the body schema of this documentation: https://developer.bigcommerce.com/api-reference/7f6ba94b629d0-upload-a-theme